### PR TITLE
[REF] template-module: Add post_load hook

### PR DESCRIPTION
**Pull request by moylop260**
- [post_load info from odoo/odoo](https://github.com/odoo/odoo/blob/2779d463e80eaf903e789c29991a1faef8d07445/openerp/modules/module.py#L317-L322)

### CODE REVIEW

File: template/module/hooks.py [hunk 1]
```diff
@@ -44,3 +44,12 @@ def uninstall_hook(cr, registry):
         Database registry, using v7 api.
     """
     raise NotImplementedError
+
+
+def post_load():
+    """Loaded before any model or data has been initialized.
+
+    This is ok as the post-load hook is for server-wide
```
Comment (pedrobaeza) [inline template/module/hooks.py]: Maybe you can document here for what is this useful

Comment (moylop260) [inline template/module/hooks.py]: @pedrobaeza 
Thanks for feedback.
Added just one useful case.
(Sorry, I don't know all useful cases)

**Comment by lmignon** [comment]
:+1: Thank you @moylop260

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**